cnf-app-mac-operator/go.mod

```diff
@@ -1,15 +1,74 @@
-module github.com/rh-nfv-int/cnf-app-mac-operator
+module github.com/openshift-kni/example-cnf/tree/main/cnf-app-mac-operator
 
-go 1.13
+go 1.20
 
 require (
-	github.com/go-logr/logr v0.1.0
-	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.1.0
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
-	k8s.io/api v0.18.10
-	k8s.io/apimachinery v0.18.10
-	k8s.io/client-go v0.18.10
-	sigs.k8s.io/controller-runtime v0.6.3
-	sigs.k8s.io/kustomize/kustomize/v3 v3.8.7 // indirect
+	github.com/go-logr/logr v1.2.4
+	github.com/onsi/ginkgo/v2 v2.9.5
+	github.com/onsi/gomega v1.27.7
+	k8s.io/api v0.27.2
+	k8s.io/apimachinery v0.27.2
+	k8s.io/client-go v0.27.2
+	sigs.k8s.io/controller-runtime v0.15.0
+)
+
+require (
```

Comment:
Do you know why there is a new require section?

Reply:
no idea :/ this file is automatically created in fact